android/src/main/java/com/azendoo/reactnativesnackbar/SnackbarModule.java

```diff
@@ -73,4 +92,19 @@ public void onClick(View v) {
 
         snackbar.show();
     }
+
+    private void recursiveLoopChildren(ViewGroup view) {
```

Comment:
Yup, the modal in RN for Android are dialogs that are always displayed above the activities. It does not work for that reason.